lib/load.js

```diff
@@ -3,6 +3,18 @@ import { main } from "./main";
 
 export const load = async () => {
   const configResponse = await fetch("config.json");
+  if (!configResponse.ok) {
```

Comment:
Can you remove the loading notice? It doesn't load any more if there was an error with the config file.

I think „Try to reload“ should stay. People can try it if there was a deployment in the meantime.